help/inbox-view-in-mobile-app.md

```diff
@@ -0,0 +1,36 @@
+# Inbox view in mobile app
+
+The **Inbox** is the default view after logging in to the Zulip mobile app. It's
+a great way to get an overview of all the messages you have received, which
+combines stream messages and direct messages. There's an unread messages counter
+to the right of each direct message, stream, and topic.
+
+This view shows unread direct messages first and then all topics with unread
+messages across all your subscribed streams.
+
+The list of streams is sorted alphabetically and includes pinned streams first,
+followed by active streams. Inactive streams and muted streams and topics are
+not shown.
+
```

Comment:
How about this shorter version?

---

The **Inbox** is the default view in the Zulip mobile app. It's
a great way to get an overview of all the unmuted conversations where you have unread messages.

The **Inbox** view shows your unread direct message conversations, followed by all topics with unread
messages, grouped by stream. The list of streams is sorted alphabetically, with pinned streams at the top.